tokio/src/sync/mpsc/mod.rs

```diff
@@ -10,8 +10,13 @@
 //! is rejected and the task will be notified when additional capacity is
 //! available. In other words, the channel provides backpressure.
 //!
-//! Unbounded channels are also available using the `unbounded_channel`
-//! constructor.
+//! This module provides two variants of the channel: A bounded and an unbounded
+//! variant. The bounded variant has a limit on the number of messages that the
```

Comment:
also, how about just
```suggestion
//! This module provides two variants of the channel: bounded and unbounded.
//! The bounded variant has a limit on the number of messages that the
```